.travis.yml

```diff
@@ -36,6 +36,8 @@ matrix:
         - sudo ssh default -t 'cd /vagrant && sudo make localintegration RUNC_USE_SYSTEMD=yes'
         # same setup but with fs2 driver instead of systemd
         - sudo ssh default -t 'cd /vagrant && sudo make localintegration'
+        # rootless
+        - sudo ssh default -t 'cd /vagrant && sudo make localrootlessintegration'
```

Comment:
Just noticed it now -- we probably want to test it with systemd as well.

I.e. something like

```
# cgroupv2+systemd: test on vagrant host itself as we need systemd
- sudo ssh default -t 'cd /vagrant && sudo make localintegration localrootlessintegration RUNC_USE_SYSTEMD=yes'
# same setup but with fs2 driver instead of systemd
- sudo ssh default -t 'cd /vagrant && sudo make localintegration localrootlessintegration'
```

Or maybe it makes sense to introduce something like `localtest` as the above is starting to look horrific.

Reply:
https://github.com/opencontainers/runc/pull/2340#discussion_r412522250